ekncontent/ekncontent/eknc-query-object.c

```diff
@@ -581,6 +581,40 @@ get_title_clause2 (EkncQueryObject *self, gchar **terms)
 }
 
 static gchar *
+get_title_clause3 (EkncQueryObject *self,
```

Comment:
Maybe out of scope for this, but it might be good to clean up these `...2` and `...3` variants - I think they were only needed for backwards compatibility in xapian-bridge. At least only `get_query_clause2` and not `get_query_clause` is used as far as I can tell.

Should this really be named `get_title_clause3` since it's not used by a corresponding `get_query_clause3`?

Reply:
There are three `get_title_clause()` implementations, but only one is correct — and effectively called. I need to mop up this mess.